pandas/core/dtypes/generic.py

```diff
@@ -54,6 +54,7 @@ def _check(cls, inst):
 ABCPeriod = create_pandas_abc_type("ABCPeriod", "_typ", ("period", ))
 ABCDateOffset = create_pandas_abc_type("ABCDateOffset", "_typ",
                                        ("dateoffset",))
+ABCInterval = create_pandas_abc_type("ABCPeriod", "_typ", ("interval", ))
```

Comment:
The first argument should be "ABCInterval" instead of "ABCPeriod", right? Or am I missing something?

Reply:
@jschendel : I don't believe you are.

Reply:
@jbrockmendel : Do we need any tests for this new `ABCType` ?

Reply:
Yah typo, will fix, good catch.